community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/BuiltInProcedures.java

```diff
@@ -699,14 +722,20 @@ public BooleanResult( Boolean success )
     public class IndexResult
     {
         public final String description;
+        public final String label;
+        public final List<String> properties;
         public final String state;
         public final String type;
+        public final Map<String,String> provider;
```

Comment:
same question why this is a map and how users should react to it: f.e.: map of providers?